kafka/README.md

```diff
@@ -1,6 +1,14 @@
 # Information for confluent-kafka-go developers
 
-Whenever librdkafka error codes are updated make sure to run generate
+## Development process
+
+1. Use go1.18 (and related tooling) for development on confluent-kafka-go.
+2. Make sure to run `gofmt` and `go vet` on your code.
+3. While there is no hard-limit, try to keep your code under 80 characters.
```

Comment:
code width?

Reply:
done, used the same language as librdkafka